10up/ElasticPress
# Index posts after Woocommerce custom sort

### Description of the Change
This PR will resolve the WooCommerce custom sort while using EP


Closes #2928

### How to test the Change

- With Woocommerce and EP enabled, go to Products
- Click on "sorting"
- Reorder your products and reload
- See the ordering has been applied
- Visit the shop page and see the ordering is applied



### Changelog Entry

> Fixed - Woocommerce custom sort


### Credits

Props @MARQAS, @felipeelia 


### Checklist:
